Implementations may not deprecate a field that the interface hasn't deprecated (#1053)
graphql/graphql-spec: spec/Section 3 -- Type System.md

       2. Let {implementedFieldType} be the return type of {implementedField}.
       3. {IsValidImplementationFieldType(fieldType, implementedFieldType)} must
          be {true}.

>     6. If the `@deprecated` directive is not applied to {implementedField}, then
>        the `@deprecated` directive must not be applied to {field}.

Reviewer: ```suggestion
    6. If {field} is deprecated then {implementedField} must also be deprecated.
```
